instrumentation/rack/test/opentelemetry/instrumentation/rack/middlewares/tracer_middleware_test.rb

```diff
@@ -113,6 +113,20 @@
         end
       end
 
+      describe 'when a regexp is passed in' do
+        let(:config) { { untraced_endpoints: %r{\A/ping/?} } }
+
+        it 'traces everything' do
+          Rack::MockRequest.new(rack_builder).get('/ping', env)
+
+          ping_span = finished_spans.find { |s| s.attributes['http.target'] == '/ping' }
+          _(ping_span).wont_be_nil
```

Comment:
If I pass `untraced_endpoints = %r{\A/ping/?}`, wouldn't I expect `must_be_nil` instead?

Reply:
I've copy-pasted this from a related spec immediately above.

Reply:
You've copy-pasted the wrong spec. This is only testing "the option expects an array", whereas we need a test for "when an array of regexes is passed in", based on the "when an array is passed in" spec.